### PR TITLE
Update to Ecto 2.1.0

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: web/utils.ex [hunk 1]
```diff
@@ -126,8 +136,8 @@ defmodule HexWeb.Utils do
     do: for(elem <- list, do: binarify(elem))
   def binarify(%Version{} = version),
     do: to_string(version)
-  def binarify(%Ecto.DateTime{} = dt),
-    do: Ecto.DateTime.to_iso8601(dt)
+  def binarify(%NaiveDateTime{} = dt),
+    do: dt |> Map.put(:microsecond, 0) |> NaiveDateTime.to_iso8601()
```
Comment (wojtekmach) [inline web/utils.ex]: added this for backwards combat: Ecto.DateTime.to_iso8601 didn't display microseconds

Comment (josevalim) [inline web/utils.ex]: You probably want to do `{0, 0}`.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: web/utils.ex (hunk 1 above)
Comment (wojtekmach) [inline web/utils.ex]: that's right, thanks!

**Pull request merged**